ws-nextjs-app/pages/[service]/new_live/[id]/Header/index.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Text from '#app/components/Text';
+import styles from './styles';
+
+const Header = ({
+  title,
+  description,
+}: {
+  title: string;
+  description?: string;
+}) => {
+  return (
+    <div css={styles.backgroundColor}>
+      <div css={styles.grid}>
+        <div css={styles.labelColumn}>
+          {/* Holding styles and text for live label */}
+          <Text
```

Comment:
I realised this is a placeholder, however this mark up needs to sit inside the h1 tag....

Reply:
That makes sense - I have updated the code